Improving the documentation. (#3319)
realm/realm-js: README.md

* nvm (on Mac)
* cocoapods (on Mac)
* Android SDK 23+

> * Android NDK 21.0 (Only available on the SDK Manager on Android Studio)

Reviewer: ```suggestion
* Android NDK 21.0 (available via the SDK Manager in Android Studio / Android SDK Tools)
```

Note: On Windows the RealmJS repo should be cloned with symlinks enabled
```

> #run in elevated 

Reviewer 2: why this change.
